src/getHumanAge.test.js

```diff
@@ -7,4 +7,20 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it('should return correct value if catAge && dogAge < 15', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  it('should return correct value if 15 <= catAge && dogAge <= 28', () => {
```

Comment:
I would recommend adding test for 29 for dogAge - it's a boundary value for this interval